data-sets-tests/src/test/java/org/zowe/data/sets/tests/DataSetsGetIntegrationTest.java

```diff
@@ -32,27 +30,10 @@ public void testGetValidDataset() throws Exception {
         createDataSet(pdsRequest);
 
         try {
+            DataSet expected = DataSet.builder().name(tempDataSet).migrated(false).build();
 
-            String pattern = "yyyy/MM/dd";
-            SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
-
-            String today = simpleDateFormat.format(new Date());
-
-            DataSetAttributes expected = DataSetAttributes.builder().blockSize(pdsRequest.getBlockSize())
-                .deviceType("3390").creationDate(today).name(tempDataSet).migrated(false)
-                .dataSetOrganization(pdsRequest.getDataSetOrganization()).expirationDate("***None***")
-                .recordLength(pdsRequest.getRecordLength()).allocationUnit(pdsRequest.getAllocationUnit())
-                .recordFormat(pdsRequest.getRecordFormat()).allocatedSize(10).used(10).build();
-
-            List<DataSetAttributes> actual = getDataSets(tempDataSet).then().statusCode(HttpStatus.SC_OK).extract()
-                .body().jsonPath().getList("items", DataSetAttributes.class);
-
-            // We can't tell the value of some attributes
-            for (DataSetAttributes dataSetAttributes : actual) {
-                dataSetAttributes.setCatalogName(null);
-                dataSetAttributes.setVolumeSerial(null);
-            }
-
+            List<DataSet> actual = getDataSets(tempDataSet).then().statusCode(HttpStatus.SC_OK).extract()
```

Comment:
Please revert this change - this is testing the attributes of the created dataset are correct